pageserver/src/tenant.rs

```diff
@@ -1855,12 +1873,21 @@ impl Tenant {
             }
         }
 
+        // We cancel the Tenant's cancellation token _after_ the timelines have all shut down.  This permits
+        // them to continue to do work during their shutdown methods, e.g. flushing data.
+        tracing::info!("Cancelling CancellationToken");
```

Comment:
This will flood logging on shutdown, let's just make this a debug instead?
```suggestion
        tracing::debug!("Cancelling CancellationToken");
```